stories/Molecules/Blocks/SingleContent/singlecontent.scss

```diff
@@ -0,0 +1,71 @@
+@import '../../../assets/scss/variables';
+@import '../../../assets/scss/mixins';
+
+.singlecard {
+  animation-duration: 300ms;
+  animation-timing-function: ease-out;
+  border: 0;
+  border-top: 2px solid $color-black;
+  display: block;
+  margin-bottom: $spacing-24;
+
+  & .singlecard__head {
+    font-size: 12px;
+    font-weight: 700;
+    letter-spacing: 0.03em;
```

Comment:
@Shubham-Sharma-77 can we give the letter spacing in px

Reply:
@surbhi90 em are using in Figma.